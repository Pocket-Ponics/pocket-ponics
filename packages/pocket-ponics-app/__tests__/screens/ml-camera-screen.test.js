import React from 'react'
import { shallow } from 'enzyme'

import MLCameraScreen from '../../screens/ml-camera-screen'

describe('ML Camera Screen', () => {
	test('renders correctly', () => {
		const tree = shallow(
			<MLCameraScreen />
		).html()
		expect(tree).toMatchSnapshot()
	})
})