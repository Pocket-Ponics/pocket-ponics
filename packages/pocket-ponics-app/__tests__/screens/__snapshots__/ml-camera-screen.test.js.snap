// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ML Camera Screen renders correctly 1`] = `"<View></View>"`;
